appmap/test/test_django.py

```diff
@@ -122,19 +121,24 @@ def test_unnamed(client, events):
     assert len(events[0].message) == 0
 
 def test_exception(client, events):
-    try:
+    with pytest.raises(RuntimeError):
         client.get('/exception')
 
+        # The 1st event: HttpServerRequestEvent
```

Comment:
These checks should be move out of the `with` block. Nothing except the request should raise a `RuntimeError`.